Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM busybox
+
+RUN dir=`mktemp -d` && trap 'rm -rf "$dir"' EXIT && \
+    wget -O- http://nixos.org/releases/nix/nix-1.7/nix-1.7-x86_64-linux.tar.bz2  | bzcat | tar x -C $dir && \
```

Comment:
This needs to be updated for Nix 1.8. Also, why not simply use the install procedure `curl https://nixos.org/nix/install | sh`?